source/config/profileUpgrader.py

```diff
@@ -0,0 +1,67 @@
+# -*- coding: UTF-8 -*-
+#A part of NonVisual Desktop Access (NVDA)
+#Copyright (C) 2016 NV Access Limited
+#This file is covered by the GNU General Public License.
+#See the file COPYING for more details.
+
+from logHandler import log
+from configSpec import latestSchemaVersion, confspec
+from configobj import flatten_errors
+from copy import deepcopy
+import profileUpgradeSteps
+
+schemaVersionKey = "schemaVersion"
+
+def upgrade(profile, validator):
+	# when profile is none or empty we can still validate. It should at least have a version set.
+	log.info("Checking schema version of config:\n%s" % profile)

```

Comment:
This is the one that prints out the profile / config in full before the upgrade process starts. This should stay as `info`.